src/main/java/home_works/home_work5/task2/Main.java

```diff
@@ -0,0 +1,12 @@
+package home_works.home_work5.task2;
+
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] numbers = {1, 5, 10, 45, 12, 3};
+        Convert convert = new Convert();
+        List<Integer> newList = convert.toList(numbers);
```

Comment:
bad var name newList  I can see that it is the list from the class List and name need to represent the entity